lib/active_model_serializers/adapter/base.rb

```diff
@@ -54,25 +54,24 @@ def include_meta(json)
         json
       end
 
-      def default_key_transform
-        :unaltered
-      end
+      class << self
+        def default_transform
```

Comment:
I just mean something real quick for devs browsing the code, in case they haven't discovered that we do key transforms yet